packages/nuxt/src/module.ts

```diff
@@ -60,7 +60,13 @@ function findDefaultSdkInitFile(type: 'server' | 'client'): string | undefined {
 
   const cwd = process.cwd();
   const filePath = possibleFileExtensions
-    .map(e => path.resolve(path.join(cwd, `sentry.${type}.config.${e}`)))
+    .map(e =>
+      path.resolve(
+        type === 'server'
+          ? path.join(cwd, 'public', `instrument.${type}.${e}`)
+          : path.join(cwd, `sentry.${type}.config.${e}`),
+      ),
```

Comment:
Is there a particular reason we're changing this?

Reply:
Yes, the file is added with `--import` and therefore needs to be added to the build as-is. This is why it is added to the `public` folder. And I renamed it to `instrument` as this got kinda the convention for the Sentry server-side setups...or what do you think? Should the file still be called `sentry.server.config.ts`?

Reply:
Have you considered how to deploy this on vercel? I'm in a similar boat with the solidstart sdk and have not yet found a good solution for how to actually `--import` the file.

Reply:
No I haven't but good point! I will also have to check this out.